types/transaction.go

```diff
@@ -108,7 +120,7 @@ func NewTransactionFromDeploy(deploy Deploy) Transaction {
 	// Use StandardPayment as true only for payments without explicit `payment amount`
 	var standardPayment = paymentAmount == 0
```

Comment:
should we check module bytes is nil?
    var standardPayment = paymentAmount == 0 && deploy.Payment.ModuleBytes == nil